src/System.Private.ServiceModel/tools/IISHostedWcfService/App_code/WcfUserNameService.cs

```diff
@@ -20,7 +20,7 @@ public override void Validate(string userName, string password)
                 throw new ArgumentNullException();
             }
 
-            if (!(userName == "test1" && password == "test1pwd") && !(userName == "test2" && password == "test2pwd"))
+            if (!(userName == "test1" && password == "Mytestpwd1") && !(userName == "test2" && password == "test2pwd"))
```

Comment:
Oh, here's the 2nd place where this string is defined.  Not  blocking for this PR, but we really need to find a way to stop duplicating code between the client and server projects.  Just something for future cleanup, not needed now.